packages/wrangler/src/__tests__/helpers/mock-oauth-flow.ts

```diff
@@ -100,11 +108,17 @@ export const mockOAuthFlow = () => {
 			expected: `https://${domain}/oauth2/token`,
 		};
 		msw.use(
-			rest.post(outcome.expected, async (req, res, ctx) => {
-				// TODO: update Miniflare typings to match full undici Request
-				outcome.actual = req.url.toString();
-				return res.once(ctx.json(makeTokenResponse(respondWith)));
-			})
+			http.post(
+				outcome.expected,
+				async ({ request }) => {
+					const url = new URL(request.url);
+
+					// TODO: update Miniflare typings to match full undici Request
+					outcome.actual = url.toString();
```

Comment:
Is this still needed?

Reply:
I'm not sure what the TODO was referring to, so I've removed it